dandiapi/api/tests/test_stats.py

```diff
@@ -33,10 +36,23 @@ def test_stats_published(api_client, published_version_factory):
 
 @pytest.mark.django_db()
 def test_stats_user(api_client, user):
+    # Reset the User table for test
+    User.objects.all().delete()
+
+    # Create users with different statuses
+    approved_user_count = 0
+    for status in UserMetadata.Status.choices:
+        status_value = status[0]
+        user = User.objects.create(username=f'{status_value.lower()}_user')
+        UserMetadata.objects.create(user=user, status=status_value)
+        if status_value == UserMetadata.Status.APPROVED:
+            approved_user_count += 1
```

Comment:
I think this would be improved if it created multiple users of each type (keeping track of how many are `APPROVED`), rather than just one user of each status type.

(This is connected to my comment in the line below, where the `== 1` assertion was retained.)

Reply:
Updated